tests/data/setup.py

```diff
@@ -14,6 +14,9 @@
 requirements = [
     'click>=5.0.0',
     'mock>=1.3.0',
+    'invenio-db[versioning]>=1',
+    'invenio-db[versioning,mysql]>=1',
```

Comment:
Yes, but now you see and are able to reproduce the issue, and most probably, the root of it is the same as the real-life example (still investigating).